### ISSUE TITLE
Benchmark: Conv Net - Accuracy: 90.26%

### ISSUE THREAD

**Issue by AbhirajHinge** [zalandoresearch/fashion-mnist#87]
Network is as follows:

- No pre-processing.

- Convolutional layer with 16 feature maps of size 5 x 5 with ELU activation.

- Max Pooling layer of size 2 x 2.

- Convolutional layer with 32 feature maps of size 5 x 5 with ELU activation.

- Max Pooling layer of size 2 x 2.

Accuracy achieved on Test Dataset is 90.26 %.
I know that CNN networks of 2 layers are already present in benchmarks, but those have been implemented in Keras and Tensorflow. This network has been implemented in PyTorch, if that counts.

The code can be found [here](https://github.com/AbhirajHinge/CNN-with-Fashion-MNIST-dataset).
